vhost-device-console/README.md

```diff
@@ -83,15 +93,20 @@ For testing the device the required dependencies are:
 The daemon should be started first:
 ```shell
 host# vhost-device-console --socket-path=/tmp/console.sock --socket-count=1 \
-                           --tcp-port=12345 --backend=network
+                           --tcp-port=12345 --backend=network # for network backend
+or
+
+host# vhost-device-console --socket-path=/tmp/console.sock --socket-count=1 \
+                           --uds-path=/tmp/vm.sock --backend=uds # for uds backend
 ```
 >Note: In case the backend is "nested" there is no need to provide
-       "--socket-count" and "--tcp-port" parameters.
+       "--socket-count", "--tcp-port" and "--uds-path" parameters.
 
 The QEMU invocation needs to create a chardev socket the device can
 use to communicate as well as share the guests memory over a memfd.
 
-There are two option for running QEMU with vhost-device-console:
+There are two option for running QEMU with vhost-device-console,
```

Comment:
Here we are talking about QEMU, please add a new section for UML (and maybe also one for QEMU with the other info we already have), it's a bit confusing.

Reply:
Add a new configuration section for UML. No changes are needed for QEMU, as QEMU does not recognize this parameter.